eessi_bot_job_manager.py

```diff
@@ -296,6 +303,14 @@ def process_new_job(self, new_job):
         """
         job_id = new_job["jobid"]
 
+        # processing placeholders in scontrol command which is defined in the bot's app.cfg (setting `scontrol_command`)
```

Comment:
Please give an example of what might be in the `scontrol_command` that requires replacing. And what would be a typical value of `new_job` so that this replacement would work. Maybe also add in which scenario this is useful - at first sight, one would just expect this to be the path to your site's `scontrol` command, why would that need placeholders?